### PR TITLE
Define new Dark Cluster configs in d2 PropertyKeys

### PR DESCRIPTION
**Pull request by lesterhaynes**

### CODE REVIEW

**Review by rickzx — COMMENTED** [review]
We also need to modify toConfig() in DarkClustersConverter.java. It translates raw map back to DarkClusterConfigMap class.

File: d2/src/main/java/com/linkedin/d2/balancer/config/DarkClustersConverter.java [hunk 1]
```diff
@@ -61,6 +61,24 @@
...
+        if (darkClusterConfig.hasDispatcherOutboundTargetRate())
+        {
+          prop.put(PropertyKeys.DARK_CLUSTER_OUTBOUND_TARGET_RATE,
+                   darkClusterConfig.getDispatcherOutboundTargetRate().toString());
+        }
+
+        if (darkClusterConfig.hasDispatcherMaxRequestsToBuffer())
+        {
+          prop.put(PropertyKeys.DARK_CLUSTER_MAX_REQUESTS_TO_BUFFER,
+                   darkClusterConfig.getDispatcherMaxRequestsToBuffer());
```
[the remark this answers is not in the document]
Comment (lesterhaynes) [reply d2/src/main/java/com/linkedin/d2/balancer/config/DarkClustersConverter.java:73]: Thanks, I'll get this back to you soon.

[the remark this answers is not in the document]
Comment (lesterhaynes) [reply d2/src/main/java/com/linkedin/d2/balancer/config/DarkClustersConverter.java]: Ready to go

**Review by rickzx — APPROVED** [review]
Looks good! Thanks for making the change

**Pull request merged**